### PR TITLE
[Components with more than 1 service] LPS-188896 Refactor XLIFFInfoFormTranslationImporter component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-188896

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Review by RenanUmV — APPROVED** [review]

File: modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java [hunk 1]
```diff
@@ -83,604 +32,27 @@
...
-	@Reference
-	private LayoutLocalService _layoutLocalService;
-
-	@Reference
-	private SAXReader _saxReader;
-
-	@Reference
-	private SegmentsExperienceLocalService _segmentsExperienceLocalService;
+	@Reference(
+		target = "(component.name=com.liferay.translation.internal.importer.XLIFFTranslationSnapshotProvider)"
```
Comment (dantewang) [inline modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java:54]: @Valmir-unicap Based on other reference (ImportTranslationMVCActionCommand) to `TranslationSnapshotProvider`, we can assume that it has only 1 service. So this reference can be without the `target` part.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Review by RenanUmV — APPROVED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
Semantic versioning issue:
```
     [exec] > Task :apps:translation:translation-api:baseline
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.translation.snapshot                   MAJOR      1.0.0      1.0.0      2.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   interface  com.liferay.translation.snapshot.TranslationSnapshotProvider
     [exec] 		-   method     getTranslationSnapshot(long,com.liferay.info.item.InfoItemReference,java.io.InputStream)
     [exec] 			-   access     abstract
     [exec] 			-   return     com.liferay.translation.snapshot.TranslationSnapshot
     [exec] 		+   method     getTranslationSnapshot(long,com.liferay.info.item.InfoItemReference,java.io.InputStream,boolean)
     [exec] 			+   access     abstract
     [exec] 			+   return     com.liferay.translation.snapshot.TranslationSnapshot
     [exec] [Baseline Warning] Bundle Version Change Recommended: 10.0.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.translation.api-9.2.1.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.translation.api/9.2.0/ce44b25b1a07bd6f67f814fa2b942aaf3076375d/com.liferay.translation.api-9.2.0.jar
```
Full console:
https://testray.liferay.com/reports/production/logs/2023-07/test-1-39/test-portal-acceptance-pullrequest(master)/5966/modules-semantic-versioning-jdk8/0/5/jenkins-console.txt.gz?authuser=0

File: modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java [hunk 2]
```diff
@@ -83,604 +32,25 @@
...
-	@Override
-	public TranslationSnapshot getTranslationSnapshot(
-			long groupId, InfoItemReference infoItemReference,
-			InputStream inputStream)
-		throws IOException, PortalException {
-
-		return _getTranslationSnapshot(
-			groupId, infoItemReference, inputStream, true);
-	}
+	implements TranslationInfoItemFieldValuesImporter {
```
Comment (dantewang) [inline modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java:38]: @Valmir-unicap Please rename this class to match the name of the interface, we usually follow the pattern that for interface `A`, we name an implementation `*A`.

Here, `XLIFFTranslationInfoItemFieldValuesImporter`.

**Review by dantewang — CHANGES_REQUESTED** [review]
@Valmir-unicap Please see my comment, fix sem ver, and resend, thanks!

**Pull request closed without merging**